src/main/java/space/obminyashka/items_exchange/dao/RoleRepository.java

```diff
@@ -1,14 +1,23 @@
 package space.obminyashka.items_exchange.dao;
 
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 import space.obminyashka.items_exchange.model.Role;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import javax.transaction.Transactional;
 import java.util.Optional;
 import java.util.UUID;
 
 @Repository
 public interface RoleRepository extends JpaRepository<Role, UUID> {
 
     Optional<Role> findByName(String name);
+
+    @Transactional
+    @Modifying
+    @Query("update User set role.id =(select id from Role where name='ROLE_USER') where username = :userName")
+    void updateUserRole(String userName);
```

Comment:
1. The method's name doesn't fit its behavior. I propose to rename it to **setUserRoleToUserByUsername**
2. "username" is the word in lowercase